lib/tasks/import/hits.rake

```diff
@@ -40,6 +40,6 @@ namespace :import do
   desc 'Import hits from S3 files in a W3C log format'
   task :from_w3c_files, [:bucket] => :environment do |_, args|
     bucket = args[:bucket]
-    IngestW3cLogs.perform_async(bucket)
+    IngestW3cLogWorker.new.perform(bucket)
```

Comment:
Is there still such a thing as `IngestW3cLogs`? Is it still used?

Reply:
nope, this is from before a rename, I think.

Reply:
Always a good sign. Probably it would be a good idea to write some rake tests in here, too.